MyAddin/MyAddinClass.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+
+namespace MyAddin
+{
+    public class MyAddinClass
+	{
+		// define menu constants
+		const string menuHeader = "-&GenericFTS";
+
+        /// Called Before EA starts to check Add-In Exist
+        public String EA_Connect(EA.Repository Repository)
+		{
+			//No special processing required.
+			return "a string";
+		}
+
+		/// Called when user Clicks Add-Ins Menu item from within EA.
+		/// Populates the Menu with our desired selections
+		public object EA_GetMenuItems(EA.Repository Repository, string Location, string MenuName)
+		{
+
+            switch (MenuName)
```

Comment:
switch/case for only 1 case? normally we use switch/case for multuple cases and a default case